packages/rolldown/src/cli/index.ts

```diff
@@ -58,4 +54,4 @@ function parseArgs(args: Record<string, any>) {
   return { configPath }
 }
 
-runMain(main)
+runMain(main, { showUsage })
```

Comment:
@pi0
when we pass a custom `showUsage` to `runMain`, the `showUsage` and `renderUsage` that `citty` has are not tree-shaken. 😞

Reply:
`runMain` is a relatively simple wrapper for convenience of mainly built-in usage. Requiring to always pass a `showUage` method makes citty DX harder. you can copy it entirely if you like but as mentioned before, you can instead contribute to upsteam's showUsage also to make it better instead of forking it.

Reply:
Okay, I'll try to contribute it!